lib/paperclip/deprecations.rb

```diff
@@ -0,0 +1,34 @@
+require "active_support/deprecation"
+
+module Paperclip
+  class Deprecations
```

Comment:
My version is not taking into account the stubs, please ignore those changes. Thank you!